.github/workflows/wheels.yml

```diff
@@ -49,49 +179,55 @@ jobs:
   build_sdist:
     name: Build source distribution
     runs-on: ubuntu-latest
+
     steps:
-      - uses: actions/checkout@v2
 
-      - uses: actions/setup-python@v2
-        name: Install Python
+      - name: Acquire sources
+        uses: actions/checkout@v2
+
+      - name: Install Python
+        uses: actions/setup-python@v2
         with:
-          python-version: '3.7'
+          python-version: '3.8'
 
-      - name: Install APT packages
+      - name: Install Debian packages
         if: contains(${{ matrix.os }}, 'ubuntu')
         run: |
-          sudo apt install libbz2-dev libhdf5-serial-dev liblzo2-dev
+          sudo apt install libblosc-dev libbz2-dev libhdf5-dev liblz4-dev liblzo2-dev libsnappy-dev libzstd-dev zlib1g-dev
           sudo apt install latexmk texlive-fonts-recommended texlive-latex-recommended texlive-latex-extra texlive-plain-generic
 
       - name: Install dependencies
         run: |
-          python -m pip install --upgrade setuptools pip wheel
-          python -m pip install -r requirements.txt
-          python -m pip install cython
-          python -m pip install sphinx>=1.1 sphinx_rtd_theme numpydoc ipython
+          pip install --upgrade setuptools pip wheel
+          pip install --requirement=requirements.txt --requirement=requirements-dev.txt
+          pip install --requirement=requirements-docs.txt
 
       - name: Build sdist
-        run: make PYTHON=python dist
+        run: python setup.py sdist
```

Comment:
Please not that ``python setup.py sdist`` is not the same of ``make dist``.
The latter also includes in the sdist tarball a pre-built version of the sphinx documentation.